framework/src/kernels/MassLumpedTimeDerivative.C

```diff
@@ -30,7 +30,7 @@ validParams<MassLumpedTimeDerivative>()
 }
 
 MassLumpedTimeDerivative::MassLumpedTimeDerivative(const InputParameters & parameters)
-  : TimeKernel(parameters), _u_dot_nodal(_var.nodalValueDot())
+  : TimeKernel(parameters), _u_dot_nodal(_var.dofValuesDot())
```

Comment:
Again - this looks much better.  In the short term `_u_dot_nodal` is fine - but hopefully we can change it in the future.